Makefile

```diff
@@ -217,9 +217,17 @@ bundle: manifests kustomize operator-sdk
 	$(KUSTOMIZE) build config/manifests | $(OPERATOR_SDK) generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
 	$(OPERATOR_SDK) bundle validate ./bundle
 
+##@ Bundle Creation Addition
+## Some addition to bundle creation in the bundle
+.PHONY: bundle-update
+bundle-update: ## Update containerImage and createdAt
+	sed -r -i "s|containerImage: .*|containerImage: $(IMG)|;" ./bundle/manifests/$(OPERATOR_NAME).clusterserviceversion.yaml
+	sed -r -i "s|createdAt: .*|createdAt: \"`date '+%Y-%m-%d %T'`\"|;" ./bundle/manifests/$(OPERATOR_NAME).clusterserviceversion.yaml
+	$(OPERATOR_SDK) bundle validate ./bundle
```

Comment:
Doing the below change will call for `operator-sdk` as part of `bundle` target, otherwise `bundle-build` and `bundle-update` will fail in this line.
I got the following error when I have run it locally:

> /home/oraz/Documents/dragonfly/medik8s/machine-deletion-remediation/bin/operator-sdk/v1.25.1/operator-sdk bundle validate ./bundle
> /bin/bash: line 1: /home/oraz/Documents/dragonfly/medik8s/machine-deletion-remediation/bin/operator-sdk/v1.25.1/operator-sdk: No such file or directory
> make: *** [Makefile:226: bundle-update] Error 127